### PR TITLE
Added documentation for Appointment ServiceCategory

### PR DESCRIPTION
**Pull request by lalitha94**
Description
----
Added the documentation for appointment.serviceCategory

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screen Shot 2021-03-15 at 11 55 00 AM
Screen Shot 2021-03-15 at 11 55 38 AM
Screen Shot 2021-03-15 at 11 56 04 AM

### CODE REVIEW

File: lib/resources/r4/appointment.yaml [hunk 1]
```diff
@@ -25,6 +25,23 @@ fields:
     }
   note: When creating a new appointment, the status must be `booked` or `proposed`.
 
+- name: serviceCategory
+  required: 'No'
+  type: CodeableConcept
+  description: A broad categorization of the service that is to be performed during this appointment.
+  binding:
+    description:  Additional details about the categorization of the service.
```
Comment (lalitha94) [inline lib/resources/r4/appointment.yaml:33]: Not sure on the description here. Any thought?

**Review by fpivonka — APPROVED** [review]

File: lib/resources/example_json/r4_examples_appointment.rb [hunk 2]
```diff
@@ -19,6 +19,25 @@ module Resources
                  'Physician - Primary Care Cerner, Patient: SMART, NANCY, Location: MX Clinic 1</p></div>'
         },
         'status': 'booked',
+        'serviceCategory': [
+          {
+            'coding': [
+              {
+                'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/16127',
```
[the remark this answers is not in the document]
Comment (lalitha94) [reply lib/resources/example_json/r4_examples_appointment.rb]: Thanks for catching. Changed it here https://github.com/cerner/fhir.cerner.com/pull/585/commits/44327297a132f702c87a8830a5f1aec6daf75391

**Review by zxeleven — APPROVED** [review]

File: lib/resources/r4/appointment.yaml (hunk 1 above)
Comment (zxeleven) [reply lib/resources/r4/appointment.yaml:33]: This description sounds good to me

File: lib/resources/r4/appointment.yaml [hunk 3]
```diff
@@ -25,6 +25,23 @@ fields:
...
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: https://hl7.org/fhir/R4/snomedct.html
+    - display: appointmentReason
+      system: http://terminology.hl7.org/CodeSystem/v2-0276
+      info_link: https://terminology.hl7.org/2.1.0/CodeSystem-v2-0276.html
+    - display: Millennium Scheduling Process Options
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/16127
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-16127-scheduling-process-options
```
Comment (praneethk2094) [inline lib/resources/r4/appointment.yaml:43]: should this be 'code-set-16127-scheduling-processing-options'? wonder if this link works! Cqn we double check please.

Comment (lalitha94) [reply lib/resources/r4/appointment.yaml]: Good catch https://github.com/cerner/fhir.cerner.com/pull/585/commits/eda9cc99bab0fe6f02a2c16f0400e236b74de0f6

**Review by praneethk2094 — APPROVED** [review]

**Comment by BS062721** [comment]
Verified on live
Screen Shot 2021-03-18 at 3 35 45 PM
Screen Shot 2021-03-18 at 3 35 59 PM

**Pull request merged**